packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -113,15 +130,22 @@ export const BootstrapCommand: ICommand = {
       { prefix: '[vscode]', debug: false }
     );
 
-    // Build typescript references
     await spawnStreaming(
       process.execPath,
-      ['scripts/build_ts_refs', '--ignore-type-failures', '--info'],
+      ['scripts/build_ts_refs', '--ignore-type-failures'],
       {
         cwd: kbn.getAbsolute(),
         env: process.env,
       },
       { prefix: '[ts refs]', debug: false }
     );
+
+    // send timings
+    await reporter.timings({
+      upstreamBranch: kbn.kibanaProject.json.branch,
+      // prevent loading @kbn/utils by passing null
+      kibanaUuid: kbn.getUuid() || null,
+      timings: timings.map((t) => ({ group: 'bootstrap', ...t })),
```

Comment:
These will end up as separate enries in ES?

Reply:
Correct